src/wdio/services/TerraService.js

```diff
@@ -64,4 +64,21 @@ export default class TerraService {
     /* Set the viewport size before the spec begins.  */
     viewportHelpers.setViewport(global.browser.options.formFactor);
   }
+
+  // To more passively support code splitting in terra dev site, wait for data to load before progressing with the test.
+  // eslint-disable-next-line class-methods-use-this
+  afterCommand(commandName) {
+    if (commandName === 'refresh' || (commandName === 'url')) {
+      if (global.browser.isExisting('[data-terra-dev-site-loading]')) {
+        try {
+          global.browser.waitUntil(() => (
+            global.browser.isExisting('[data-terra-dev-site-content]')
+          ), 5000, '', 100);
```

Comment:
Thoughts on updating this to be `global.browser.options. waitforTimeout`. We currently set this value to 3000

Reply:
I had it set to that but saw some random timeouts and bumped the value up. There isn't harm in increasing the timeout as this only will affect dev site code split examples.